### ISSUE TITLE
`\subseteq` in concealed mode

### ISSUE THREAD

**Issue by jmmaloney4** [artisticat1/obsidian-latex-suite#40]
When I am typing `\subseteq`, the editor always inserts a space between `\subset` and `eq`, and it shows as `⊂eq` in when editing display blocks. I see `\subseteq` here: https://github.com/artisticat1/obsidian-latex-suite/blob/5f89fd0f63605bbde948a3bd41cc9e84a860bca6/src/conceal_maps.ts#L145 but something about the matching seems to be messed up.

**Comment by artisticat1** [comment]
Ah, this is because `\subseteq` is missing from the list of symbols in the `${SYMBOL}` variable. I'll fix it for the next release.

In the meantime, you can create a snippet for `\subseteq` and place it at the top of your snippets or give it a higher priority.

**Comment by artisticat1** [comment]
My mistake, this is just because "subset" is a substring of "subseteq" so it gets triggered before subseteq. You can use this snippet

```typescript
{trigger: "\\subset eq", replacement: "\\subseteq", options: "mA"},
```

to work around this. It's included in v1.4.1 by default. I also fixed the conceal of this symbol, so it now shows properly as ⊆!
